packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -399,45 +431,31 @@ export default class ImageEdit implements EditorPlugin {
 
     /**
      * Get image wrapper from image
-     * @param image The image to get wrapper from
      */
-    private getImageWrapper(image: HTMLImageElement): HTMLElement {
-        // Get the image wrapper from image using Entity API
-        const entity = getEntityFromElement(image?.parentNode?.parentNode as HTMLElement);
-
-        return entity?.type == IMAGE_EDIT_WRAPPER_ENTITY_TYPE ? entity.wrapper : null;
+    private getImageWrapper(): HTMLElement {
+        return document.getElementById(this.imageId);
```

Comment:
So the cloned image will have the same id with the original one, how can we guarantee this will return the cloned one?

And if there are 2 editors in the same document, it is possible two editors both have image with the same id. Better change the image id to combine editor id and image id to make it unique

Reply:
And instead of querying by image id, since there will be only one image editing per editor, you can just hold the reference to the wrapper element in this plugin. So it will be easier to get it